src/utils/db.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+const utils = require('util');
+
+global.TextEncoder = utils.TextEncoder;
```

Comment:
The assignment `global.TextEncoder = utils.TextEncoder;` is unnecessary for Sequelize operations and can be removed unless there is a specific use case for it.